### PR TITLE
To do list testing

### PR DESCRIPTION
**Pull request by ChaabaneAnas**
## Changes 

- jest Mock localStorage
- Add Tests for the adding function Using jest
- Add Tests for the deleting function Using jest

### CODE REVIEW

File: __tests__/store.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    expect(containerElem.children.length).toBe(2);
+  });
+
+  test('Number of items should be 2 ', () => {
+    document.querySelector('button[id="3"]').click();
+    Store.deleteTask(3);
+    const containerElem = document.querySelector('.todo-container');
+    expect(containerElem.children.length).toBe(1);
+  });
+});
```
Comment (rotimiazeez) [inline __tests__/store.test.js]: - Please make sure all your tests are passing so you can go to the next project 👍🏽 
Screenshot from 2022-08-19 20-48-14

Screenshot from 2022-08-19 20-48-50

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team 👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: __tests__/store.test.js (hunk 1 above)
Comment (ChaabaneAnas) [reply __tests__/store.test.js:74]: Hello @rotimiazeez
The test are fine i think u missed installing jsdom that’s why 
Ur vs code cannot read dom functions

Comment (ChaabaneAnas) [reply __tests__/store.test.js:74]: Please install jsdom and try running the tests again

**Review by victorokoroji — APPROVED** [review]
Hi @ChaabaneAnas , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- All tests are passing ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**